virttest/virt_vm.py

```diff
@@ -1206,33 +1208,28 @@ def wait_for_serial_login(self, timeout=LOGIN_WAIT_TIMEOUT,
         :params virtio: is a virtio console.
         :return: A ShellSession object.
         """
-        error_messages = []
         logging.debug("Attempting to log into '%s' via serial console "
                       "(timeout %ds)", self.name, timeout)
-        end_time = time.time() + timeout
+        end_time, exception = time.time() + timeout, None
         while time.time() < end_time:
             try:
```

Comment:
Yes, That's why I ask to reset exception to None.
If the "N-1" time failed and set exception, but "N" time success,
the exception is not correct.
 I'm right?